docs/reST/ref/key.rst

```diff
@@ -343,6 +343,9 @@ for ``KMOD_NONE``, which should be compared using equals ``==``). For example:
    corresponding key constant exists and is unique). If the return value is
    passed to the ``key_code`` function, the original constant will be returned.
 
+   **Experimental:** use_compat feature is still in development and will change.
```

Comment:
Not sure why this specific note has "will change" while everything else has "may change". I assume it's accidental, because anyone has not complained about this addition yet (unless I missed something)

Reply:
Thanks. This was a mistake.